### DISCUSSION TITLE
build的时候，报TS2300: Duplicate identifier 'event'.

### DISCUSSION THREAD

**Discussion by ibabamu** [element-plus/element-plus#10721] [Q&A]
build的时候，报下面的错误，有遇到过的吗

node_modules/element-plus/es/components/tree-v2/index.d.ts:97:511 - error TS2300: Duplicate identifier 'event'

**Comment by ibabamu** [comment]
从2.2.19开始，就有这个问题，包括最新版2.2.22

**Comment by btea** [comment]
请提供一个可复现的地址。

**Reply by ibabamu** [reply]
https://github.com/makunet/maku-admin

这个是基于element-plus开发的，升级到2.2.19+，就不能build了

**Comment by chenxch** [comment]
![image](https://user-images.githubusercontent.com/23251408/207604543-43cdd4e1-ebfa-4e20-8aee-1472d2e8afc0.png)

@ibabamu 问题应该解决了吧
